.gitignore

```diff
@@ -36,3 +36,6 @@
 /.vscode
 # JetBrains IDE stuff.
 /.idea
+
+# Packaged module
```

Comment:
```suggestion
# Packaged module.
```

What is a packaged module?

Reply:
When you exec [npm pack](https://docs.npmjs.com/cli/v7/commands/npm-pack) manually and not as part of `npm publish`, it generates a tarfile, that `*.tgz` file that I've put in the `.gitignore` file. I usually do that to check what's actually being included in the package itself that's going to be published, since it's a good practice to don't add test files or similar things. The tarfile can be also served from a vanilla HTTP server or installed standalone, for example, but you don't want to upload it to the git repo since it's already a copy of the repo code.

Reply:
ok